docs/manual/src/udl/ext_types_wrapped.md

```diff
@@ -1,29 +1,55 @@
-# External types which wrap primitive types
+# External types which wrap UDL types
 
-*Note: The facility described in this document is not yet available for all foreign language
-bindings.*
+*Note: The facility described in this document is not yet available for the Ruby bindings.*
 
-It's possible to expose types in your Rust crate to UniFFI, so long as the type can
-be expressed as one of UniFFI's [builtin_types](./builtin_types.md).
+`Wrapped` types allow you to extend the UniFFI type system with custom types.
+These types rely on a "normal" UDL type move data across the FFI, then use a
+custom type to wrap those types on the scaffolding and/or bindings side.
 
-For example, consider the following trivial Rust abstraction for a "handle" which wraps an
+Some examples:
+  - You have Rust type named `Handle` which uses the new-type pattern to wrap a
```

Comment:
nit: Describing a `Handle` as wrapping a string breaks my mental model of what a handle usually represents. Is there a reason why you changed it from wrapping an int? Having something other than a `String` being described seems worthwhile.

(I just noticed the example for handle below did stick with ints)